tutorials/fit/NumericalMinimization.py

```diff
@@ -0,0 +1,85 @@
+#  \file
+#  \ingroup tutorial_fit
+#  \notebook -nodraw
+#  Example on how to use the new Minimizer class in ROOT
+#   Show usage with all the possible minimizers.
+#  Minimize the Rosenbrock function (a 2D -function)
+#  This example is described also in
+#  http://root.cern.ch/drupal/content/numerical-minimization#multidim_minim
+#  input : minimizer name + algorithm name
+#  randomSeed: = <0 : fixed value: 0 random with seed 0; >0 random with given seed
+#
+#  \macro_code
+#
+#  \author Lorenzo Moneta
+
+import ROOT
+
+def RosenBrock( vecx ):
+   x = vecx[0]
+   y = vecx[1]
+   return (y-x**2)**2 + (1-x)**2
+
+# create minimizer giving a name and a name (optionally) for the specific algorithm
+#  possible choices are:
+#      minimizerName                  algoName
+#     Minuit                     Migrad, Simplex,Combined,Scan  (default is Migrad)
+#      Minuit2                    Migrad, BFGS, Simplex,Combined,Scan  (default is Migrad)
+#     GSLMultiMin                ConjugateFR, ConjugatePR, BFGS, BFGS2, SteepestDescent
+#     GSLSimAn
+#     Genetic
+def NumericalMinimization(minimizerName = "Minuit2",
+                          algoName = "" ,
+                          randomSeed = -1) :
+
+   minimizer = ROOT.Math.Factory.CreateMinimizer(minimizerName, algoName);
+   if (not minimizer) :
+      print("Error: cannot create minimizer \"",minName,"\". Maybe the required library was not built?")
+      return 1;
+
+   # set tolerance and other minimizer parameters
+   # one can also use default values
+   minimizer.SetMaxFunctionCalls(1000000); # working for Minuit/Minuit2
+   minimizer.SetMaxIterations(10000);  # for GSL minimizers - no effect in Minuit/Minuit2
+   minimizer.SetTolerance(0.001);
+   minimizer.SetPrintLevel(1);
+
+   # create function wrapper for minimizer
+
+   f = ROOT.Math.Functor(RosenBrock,2);
+
+   #evaluate function at a point
```

Comment:
Comment should start with capital letter. Also, there should be a whitespace after the `#` sign